lib/wallaby/browser.ex

```diff
@@ -857,6 +857,16 @@ defmodule Wallaby.Browser do
     end
   end
 
+  defp do_get(query, elements) do
+    getNumber = query.conditions[:get]
+    case {getNumber, length(elements)} do
```

Comment:
for variable names we prefer `snake_case` over `camelCase` (maybe that's not camel but something differnet, you know what I'm gettign at) :)